starter-code/index.html

```diff
@@ -83,8 +83,9 @@
     <script src="/vendor/scripts/marked.js"></script>
     <script src="/scripts/article.js"></script>
     <script src="/scripts/articleView.js"></script>
-    <script>
+    <script src"/scripts/article.js">
       // TODO: What model function do we need to call, to initiate the retrieval of data?
+      Article.fetchAll();
```

Comment:
Article.fetchAll() needs its own script tag. You can't combine it with the script tag for article.js. When do so, none of your script file will run, and nothing will render.